.github/workflows/release.yml

```diff
@@ -51,8 +54,16 @@ jobs:
         set -eo pipefail
         echo podspec file verification starts:
         make test-podspec
+        
+    - name: Build Artifacts
+      if: ${{ github.event.inputs.action_tag == 'yes' }}
+      run: |
+        make archive
+        make zip
 
-    - uses: release-drafter/release-drafter@v5
+    - name: Create GH Release
+      id: create_release
+      uses: release-drafter/release-drafter@v5
```

Comment:
Core is using this specific commit, `uses: release-drafter/release-drafter@fe52e97d262833ae07d05efaf1a239df3f1b5cd4`. Can you double check if this is required or if using `v5` is correct?

Reply:
Core reverted to v5.